README.md

```diff
@@ -24,6 +24,15 @@ cd bsd
 yarn install
 ```
 
+### Troubleshooting
+
+You may need to install:
+
+```
+sudo apt install -y build-essential
+sudo apt install -y libpng-dev libjpeg-dev
```

Comment:
These are really for `module-scan`, right?

Reply:
Yes 😑